docs/sdk/client-side-sdks/javascript.md

```diff
@@ -249,6 +239,35 @@ dvcClient.flushEvents(() => {
 });
 ```
 
+### Tracking Events Example
+
+For our example, we will be querying for a checkout button using it’s ID selector, and adding a click event listener to trigger our DevCycle event tracking.
+
+```javascript
+const checkoutButton = document.querySelector("#checkout-button");
+checkoutButton.addEventListener("click", () => {
+  const event = {
+    type: "checkout_button_clicked", // this is required
+    metaData: {
+      page: "homepage",
+    },
+  };
+  dvcClient.track(event);
+});
+```
+
+Next, we'll need to set up the event metrics within our experiment on the DevCycle Dashboard. To do so, we'll need to navigate to the `Comparative Analysis` section of our feature.
+
+![Comparative Analysis section within a feature on the DevCycle Dashboard](/dvc-comparative-analysis.png)
+
+From here, we create a new metric or select one that has already been created. Since we don't currently have any metrics, we'll have to create one.
+
+Let's set one up for the home page checkout button in our example above.
+
+![DevCycle Create a New Metric pop-up modal for setting up metrics to track events](/dvc-create-new-metric.png)
```

Comment:
I wouldn't put this in the JS sdk example. Should be it's own guide for Web metrics we link out to.